src/main/java/org/wildfly/swarm/plugin/utils/DescriptorUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.wildfly.swarm.plugin.utils;
+
+import org.jboss.shrinkwrap.descriptor.api.Descriptor;
+import org.jboss.shrinkwrap.descriptor.spi.node.Node;
+import org.jboss.shrinkwrap.descriptor.spi.node.NodeDescriptor;
+
+public class DescriptorUtils {
```

Comment:
This class is only used in `ModuleRewriteRules` but if you prefer to keep it standalone (may be other classes will refer to it as well) then it is fine as well

Reply:
Wanted to extract that code from the class, as it has a lot of inner classes.